### PR TITLE
Update zync dashboards

### PR DESCRIPTION
**Pull request by raelga**
This PR will update the operator with the new zync dashboard.

- Adds new rails metrics
- Adds new queue metrics
- Adds new pod metrics
- Removes the zync-que dashboard, as now is included in the main dashboard

### CODE REVIEW

File: pkg/assets/assets/monitoring/zync-grafana-dashboard-1.json.tpl [hunk 1]
```diff
@@ -14,21 +14,2296 @@
...
+      "renderer": "flot",
+      "repeat": null,
+      "repeatDirection": null,
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "kube_deployment_status_replicas{namespace='$namespace',deploymentConfig=~'$deploymentConfig'}",
```
Comment (slopezz) [inline pkg/assets/assets/monitoring/zync-grafana-dashboard-1.json.tpl:1764]: All the queries on that part has not been updated to replicationcontroller metrics.

Can you please fix them?

Comment (raelga) [reply pkg/assets/assets/monitoring/zync-grafana-dashboard-1.json.tpl]: Solved in https://github.com/3scale/3scale-operator/pull/410/commits/fd9ff9e6e61a5cb7fd6bdef76f522fb12e8c3dd4 

https://grafana-route-application-monitoring.apps.dev-eng-ocp4-3.dev.3sca.net/d/fGWq6GGGz/slopez-example-3scale-zync

**Comment by eguzki** [comment]
unittests failed. Let me fix for you

**Comment by raelga** [comment]
> unittests failed. Let me fix for you

Don't worry Egus, I have found the issue and I'm working on it, template not properly escaped.

**Comment by raelga** [comment]
@eguzki all green!

**Pull request merged**